documentation/src/main/asciidoc/topics/security.adoc

```diff
@@ -136,36 +136,6 @@ the DN +cn=managers,ou=people,dc=example,dc=com+ will be mapped to the role +man
 * ClusterRoleMapper (Java: +org.infinispan.security.impl.ClusterRoleMapper+ XML: +<cluster-role-mapper />+): a mapper which uses the ClusterRegistry to store principal to role mappings. This allows the use of the CLI's GRANT and DENY commands to add/remove roles to a principal.
```

Comment:
The new CLI still doesn't support GRANT/DENY, so we should remove it until I have it implemented the way we want it.

Reply:
@tristantarrant I removed the entire `ClusterRoleMapper` line not just the sentence about CLI GRANT/DENY